filebeat/harvester/log.go

```diff
@@ -292,12 +297,12 @@ func (h *Harvester) newLogFileReader() (reader.Reader, error) {
 	var r reader.Reader
 	var err error
 
-	fileReader, err := reader.NewLogFile(h.file, readerConfig, h.done)
+	logInput, err := NewLogFile(h.file, readerConfig)
```

Comment:
I have to investigate that further. It somehow also relates tot he previous point.

Questions for me:
* What happens if Close is called twice?
* Readers are closed anyways if harvester is ended. Do we to close them anyways?